services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala

```diff
@@ -149,23 +145,24 @@ class AuthServiceTest extends WordSpec with Matchers with MockFactory {
     }
 
     "responseFromUser" should {
-      "return AuthResponse" in {
-        (authUtils.getAuthResponse _ when (*, *, *)).returns(Some(any[AuthResponse]))
-        authService.responseFromUser(dummyUser) shouldBe Some(any[AuthResponse])
```

Comment:
`any[AuthResponse]` caused problems with other tests. It makes sense, but I have no idea, why this test did not fail by itself.
Mockito is great!